server/src/packets/PacketHandler.cpp

```
@@ -316,3 +317,31 @@ int packet::PlayerDisconnectedHandler::handlePacket(server::Server &server,
   server.clearClientSlot(client._player_id);
   return OK;
 }
+
+int packet::InputPlayerHandler::handlePacket(server::Server &server,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)